src/client/entity-editor/series-section/series-editor.tsx

```diff
@@ -74,7 +78,64 @@ type SeriesItemsProps = {
  * @returns {ReactElement} React Component as a modular section within the entity editor.
  *
  */
-function SeriesEditor({baseEntity, relationshipTypes, seriesType, onRemove, onAdd, onEdit, seriesItemsArray}:SeriesItemsProps) {
+
+const SeriesList = ({value, baseEntity, handleNumberAttributeChange, onRemove, dragHandler}) => (
```

Comment:
I think the name is misleading if this component is for one item rather than the list of items. Maybe SeriesItem or SeriesListItem?